test/trackler/problem_test.rb

```diff
@@ -4,25 +4,65 @@
 class ProblemTest < Minitest::Test
   def test_existing_problem
     problem = Trackler::Problem.new('hello-world', FIXTURE_PATH)
-
     assert problem.exists?, "hello world files not found in the metadata dir"
+  end
+
+  def test_slug
```

Comment:
These could be combined in a table and tested but in some ways DRY'ing the tests is often not a great idea.  It is tedious to write these out like this though, but it is highly communicative, easy to drop one out, or add one in, and easy to find when failing.